specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2024-11-01-preview/examples/CreateOrUpdateNotificationRule.json

```diff
@@ -0,0 +1,156 @@
+{
+  "operationId": "NotificationRuleAdministration_CreateOrUpdateNotificationRule",
+  "title": "Create or update a notification rule.",
+  "parameters": {
+    "api-version": "2024-11-01-preview",
+    "notificationRuleId": "12345678-1234-1234-1234-123456789012",
+    "body": {
+      "displayName": "Sample_Notification_Rule",
+      "actionGroupIds": [
+        "/subscriptions/12345678-1234-1234-1234-123456789012/resourceGroups/sampleRG/providers/microsoft.insights/actionGroups/sampleActionGroup"
+      ],
+      "scope": "Tests",
+      "testIds": [
+        "12345678-1234-1234-1234-123456789012"
+      ],
+      "eventFilters": {
+        "event1": {
+          "kind": "TestRunEnded",
+          "condition": {
+            "testRunStatuses": [
+              "FAILED",
+              "DONE"
+            ],
+            "testRunResults": [
+              "PASSED",
+              "FAILED"
+            ],
+            "createdByTypes": [
+              "User",
+              "ScheduleTrigger"
+            ]
+          }
+        },
+        "event2": {
+          "kind": "TestRunStarted",
+          "condition": {
+            "createdByTypes": [
+              "User",
+              "ScheduleTrigger"
+            ]
+          }
+        },
+        "event3": {
+          "kind": "TriggerEnded"
+        },
+        "event4": {
+          "kind": "TriggerDisabled"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "notificationRuleId": "12345678-1234-1234-1234-123456789012",
+        "displayName": "Sample_Notification_Rule",
+        "actionGroupIds": [
+          "/subscriptions/12345678-1234-1234-1234-123456789012/resourceGroups/sampleRG/providers/microsoft.insights/actionGroups/sampleActionGroup"
+        ],
+        "scope": "Tests",
+        "testIds": [
+          "12345678-1234-1234-1234-123456789012"
+        ],
+        "eventFilters": {
+          "event1": {
```

Comment:
nit: might want to change these to include filter in the name as well (though I agree this is user generated string)

Reply:
Done